account_view_adj/views/account_invoice_views.xml

```diff
@@ -5,8 +5,12 @@
         <field name="model">account.invoice</field>
         <field name="inherit_id" ref="account.invoice_tree" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='amount_total_signed']" position="after">
-                <field name="amount_untaxed_signed" sum="Total w/o Tax" />
+            <xpath expr="//field[@name='origin']" position="after">
+                <field
+                    name="amount_untaxed_signed"
+                    string="Untaxed Amount in Company Curr"
```

Comment:
I'd make the string shorter (for English users), given that this is specifically for the label in the list view. This is non-blocking though.